dist/profile/templates/confluence/vhost.conf

```diff
@@ -2998,6 +2998,8 @@ RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Features\+controlled\+by\+system\+properties$" "https://jenkins.io/doc/book/managing/system-properties/" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Groovy\+Hook\+Script$" "https://jenkins.io/doc/book/managing/groovy-hook-scripts/" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://www.jenkins.io/doc/book/managing/security/#cross-site-request-forgery" [NE,NC,L,QSA,R=301]
```

Comment:
Should follow the same URL pattern as elsewhere in the file, using `jenkins.io` instead of `www.jenkins.io`.

```suggestion
RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://jenkins.io/doc/book/managing/security/#cross-site-request-forgery" [NE,NC,L,QSA,R=301]
```

Reply:
While I totally agree that it should be set to 'www' to remove one redirection, for now, I agree with @MarkEWaite for consistency it's better to use the same base URL than the others,